access_management/permission_constants.py

```diff
@@ -28,3 +28,5 @@
     PROJECT_CONTRIBUTOR_GROUP: project_contributor_permissions,
     PROJECT_VIEW_ONLY_GROUP: project_view_only_permissions,
 }
+
+all_perms = vars()
```

Comment:
The intent was to remove the wildcard import over in `access_management.utils`, but I didn't want to significantly refactor this code since it would be out of scope for this PR. Just moved this `dict` here so that we could import elsewhere. 

The use of `vars` in `access_management.utils` was a convenient (although opaque) way of automatically constructing a `dict` that contained these different permssions/groups that could be access programmatically. In a future update we should probably move away from doing this and move a lot of this stuff over to `projects.permissions.py`

Reply:
`vars` itself just returns the `__dict__` attribute of whatever context it's called in. In this case it's retrieving the `__dict__` attribute of the module.

https://docs.python.org/3/library/functions.html#vars

Reply:
Awesome. Thank you!

Reply:
I could've really used this explanation 3 months ago.